src/endpoint/s3/ops/s3_get_object_acl.js

```diff
@@ -1,12 +1,46 @@
 /* Copyright (C) 2016 NooBaa */
 'use strict';
 
+const _ = require('lodash');
+const S3Error = require('../s3_errors').S3Error;
+const config = require('../../../../config');
 const s3_utils = require('../s3_utils');
 
 /**
  * http://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGETacl.html
  */
-function get_object_acl(req) {
+async function get_object_acl(req) {
+    if (config.ACL_HANDLING === "reject") {
+        throw new S3Error(S3Error.AccessDenied);
+    } else if (config.ACL_HANDLING === "pass-through") {
+        //TODO: form a XML from the info in request
+        const reply = await req.object_sdk.get_object_acl({
```

Comment:
This new flow seem too similar to the existing code (the next case).
Can't we merge those cases?